src/routers/controllers/stopPageAddOwnerControllerGet.ts

```diff
@@ -1,20 +1,24 @@
 import { Request, RequestHandler, Response } from "express";
 import * as constants from "../../lib/constants";
 import { getTranslationsForView } from "../../lib/utils/translationUtils";
-import { AnyRecord } from "types/utilTypes";
-import { getLoggedUserAcspMembership } from "../../lib/utils/sessionUtils";
+import { getExtraData, getLoggedUserAcspMembership } from "../../lib/utils/sessionUtils";
 import { AcspMembership } from "private-api-sdk-node/dist/services/acsp-manage-users/types";
+import { MemberForRemoval } from "types/membership";
+import logger from "../../lib/Logger";
 
 export const stopPageAddOwnerControllerGet: RequestHandler = async (req: Request, res: Response) => {
-    res.render(constants.STOP_PAGE_ADD_ACCOUNT_OWNER, getViewData(req));
-};
-
-const getViewData = (req: Request): AnyRecord => {
-
     const loggedUserAcspMembership: AcspMembership = getLoggedUserAcspMembership(req.session);
-    return {
+    const userToRemove: MemberForRemoval | undefined = getExtraData(req.session, constants.DETAILS_OF_USER_TO_REMOVE);
+
+    if (!userToRemove) {
+        logger.error(`${stopPageAddOwnerControllerGet.name}: DETAILS_OF_USER_TO_REMOVE not found in session`);
+        throw Error("DETAILS_OF_USER_TO_REMOVE not found in session");
+    }
+    res.render(constants.STOP_PAGE_ADD_ACCOUNT_OWNER, {
+        buttonHref: constants.ADD_USER_FULL_URL,
         lang: getTranslationsForView(req.t, constants.STOP_PAGE_ADD_ACCOUNT_OWNER),
         companyName: loggedUserAcspMembership.acspName,
-        linkHref: constants.MANAGE_USER_FULL_URL
-    };
+        linkHref: constants.MANAGE_USER_FULL_URL,
+        backLinkUrl: constants.REMOVE_MEMBER_CHECK_DETAILS_FULL_URL.replace(":id", userToRemove.id)
```

Comment:
What about a function instead of calling `.replace`? 

Small suggestion but with a function we can find usages a bit more easily, and would remove the `.replace` logic, while it's fine cause it's a constant - `.replace` always gives me a small worry incase it's used in a place where `:id` coincidentally appears elsewhere (obviously it doesn't here because we know what the constant is).


```javascript
export const getRemoveMemberCheckDetailsUrl = (id) => `/${REMOVE_MEMBER_PAGE}/${id}`;
const backLinkUrl = getRemoveMemberCheckDetailsUrl(userToRemove.id);
```

Reply:
thanks, good suggestion. Updated the PR.